.github/steps/4-play-time.md

```diff
@@ -1,59 +1,97 @@
+
+### ⌨️ Activity: Assign tasks through the Agents Panel :robot:
+
+> [!IMPORTANT]
+> Make sure you merged the `prepare-environment` branch from the previous step before proceeding.
+
+Let's get you familiarized with the Agents panel!
+
+1. In a new tab, open the **Copilot Agents** panel from the top navigation bar
+   <img width="600" alt="Agents panel view" src="https://github.com/user-attachments/assets/d0eba779-ab37-4cf9-9229-dca8a64c119c" />
+
+1. Make sure the `{{ full_repo_name }}` repository is selected in the panel and the branch is set to `main`.
+1. Assign Copilot to work on the following task:
 
-1. Find the open issue titled `Simplify Change requests for teachers`.
+   > ![Static Badge](https://img.shields.io/badge/-Task%201-text?style=social&logo=github%20copilot)
+   >
+   > ```prompt
+   > Integrate social sharing buttons so users can easily share activities with their friends
+   > ```
 
-1. Familiarize yourself with the issue, **Assign it to Copilot**, and switch to the created pull request.
+   > ✨ **Bonus:** Think of something you would want improved on the website and assign it to Copilot. You can use your creativity here!
 
-1. Scroll down and click the **View Session** button to watch Copilot's live progress. Alternately, go take a quick break. ☕️
+1. After a moment, you will notice that the task appears in the panel with its current status. You can check back here for a high level overview of all your assigned tasks.
 
-1. (optional) Modify the templates Copilot provided.
+   <img width="400" alt="Agents Panel task in progress view" src="https://github.com/user-attachments/assets/0fc53085-d68d-496e-a81b-1ab78014db63" />
+
+1. Click on the task to open the pull request in a new tab and track how Copilot is working on it in real time - just like you did in the first step.
+
+1. Scroll down and click the **View Session** button.
+1. In the session logs, you'll notice Copilot begins by running the customization steps you've setup in the previous step!
+   <img width="900" alt="Copilot session logs with copilot setup steps" src="https://github.com/user-attachments/assets/72c282f6-7780-4c8f-ad0b-ac8d21c4661c" />
+
+   
+1. Let's leave Copilot to work its magic for now, you can come back to review the results later ✨
+
+> [!TIP]
+> You can also access the Agents Panel in full screen mode at https://github.com/copilot/agents
 
-1. When Copilot session is finished, click the **Ready for Review** button to change the Copilot's pull request draft status.
 
-1. Merge the results. Wow, that was easy! Right?! 🥹
 
 ### ⌨️ Activity: Try implementing 2 issues simultaneously 🤯
 
+You still have some issues opened on the repository, let's see how Copilot can handle working on multiple issues at the same time!
+
 1. In the top navigation, select the **Issues** tab.
 
 1. Find the the following 2 issues and open each in a new tab.
 
    - `Difficulty Tracks`
 
-   - `Outdated Docs`
+   - `Dark Mode`
```

Comment:
explanation:
Dark mode consistently gave me nice results and screenshots in the PR using playwright MCP. 

<img width="902" height="892" alt="Image" src="https://github.com/user-attachments/assets/1be5a9a7-6978-4116-b3a2-4227155814f7" />